api/structure/schema.py

```diff
@@ -68,15 +68,19 @@ class Query(graphene.ObjectType):
     @Auth.user_roles('Admin')
     def resolve_all_structures(self, info):
         query = Structure.get_query(info)
-        return query.order_by(func.lower(StructureModel.name)).all()
+        location_id = admin_roles.user_location_for_analytics_view()
+        all_structures = query.filter(
+          StructureModel.location_id == location_id).all()
```

Comment:
Why do you use an equality operator instead of an assignment operator here?

Reply:
That's because am using the filter query and not the filter_by query to check for equality to the exact location Id